ThumbnailSequence.cpp

```diff
@@ -1560,7 +1560,7 @@ ThumbnailSequence::CompositeItem::CompositeItem(
 bool
 ThumbnailSequence::CompositeItem::incompleteThumbnail() const
 {
-	return dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
+	return m_pThumb == 0 && dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
```

Comment:
You use the following to move one std::unique_ptr to another:
ptr1 = std::move(ptr2);

Still, I think it's a compilation problem. You said you recompiled Qt, but did you verify your custom version is loaded at run time? You could verify by running scantailor under gdb and using "info shared" command to list the loaded shared libraries.